$OpenBSD$

The binaries are installed as ebc and edc, not bc and dc.

Index: include/vm.h
--- include/vm.h.orig
+++ include/vm.h
@@ -85,7 +85,7 @@
 #define BC_MAX_EXP ((unsigned long) ULONG_MAX)
 #define BC_MAX_VARS ((unsigned long) (SIZE_MAX - 1))
 
-#define BC_IS_BC (BC_ENABLED && (!DC_ENABLED || vm->name[0] != 'd'))
+#define BC_IS_BC (BC_ENABLED && (!DC_ENABLED || vm->name[1] != 'd'))
 #define BC_IS_POSIX (BC_S || BC_W)
 
 #if BC_ENABLE_SIGNALS
